modules/ztp-configuring-a-static-ip.adoc

```diff
@@ -79,12 +79,12 @@ spec:
     name: <cluster_name>
     namespace: <cluster_name>
   sshAuthorizedKey: <public_key>
-  agentLabelSelector:
-    matchLabels:
-      cluster-name: <cluster_name>
+  agentLabels: <1>
+    location: "<label-name>"
   pullSecretRef:
     name: assisted-deployment-pull-secret
   nmStateConfigLabelSelector:
     matchLabels:
       sno-cluster-<cluster-name>: <cluster_name> # Match this label
 ----
+<1> Displays labels created by the user. The labels apply when the agents boot.
```

Comment:
I think the description is misleading. It doesn't "Displays labels", It sets a label to match.

Reply:
@vkolodny Thank you, I updated it. Could you check again?

Reply:
/lgtm